lms/services/application_instance.py

```diff
@@ -310,6 +310,7 @@ def create_application_instance(  # pylint:disable=too-many-arguments
                 "canvas": {
                     "sections_enabled": False,
                     "groups_enabled": bool(developer_key),
+                    "file_picker_folders": bool(developer_key),
```

Comment:
Will this enable file picker folders automatically for all existing instances that have a developer key? Do we want to do that? I think it might break existing instances that use a scoped developer key and don't have the new scopes

Reply:
This would enable it for new installs.

Even doing that might be risky as we'd need to sync the deploy with updating KB articles etc. I'll leave as false for now and we can switch it around once we found ourselves enabling it for every new install.